.env.test

```diff
@@ -13,8 +13,7 @@ OPENSTACK_REGION='test-region'
 DEFAULT_INSTANCE_BASE_DOMAIN='example.com'
 GANDI_API_KEY='TEST_GANDI_API_KEY'
 GITHUB_ACCESS_TOKEN='test-token'
-WATCH_ORGANIZATION='test-org'
-WATCH_FORK='watched/fork'
+# WATCH_ORGANIZATION and WATCH_FORK were removed
```

Comment:
nit: no need to leave this comment, we have a git history to tell us what's been removed.